angular-workspace/projects/ni/nimble-angular/src/directives/combobox/nimble-combobox-control-value-accessor.directive.ts

```diff
@@ -64,14 +66,16 @@ export class NimbleComboboxControlValueAccessorDirective implements ControlValue
     @HostListener('blur')
     private onTouched: () => void;
 
-    public constructor(
-        private readonly _renderer: Renderer2,
-        private readonly _elementRef: ElementRef
-    ) {}
+    private readonly observer: MutationObserver;
+
+    public constructor(private readonly _renderer: Renderer2, private readonly _elementRef: ElementRef) {
```

Comment:
Can you type the `ElementRef<Combobox>` think we can then remove some `as HTMLElement` and `as Combobox` casting